org.scala-ide.sdt.core/src/org/scalaide/core/completion/CompletionProposal.scala

```diff
@@ -223,6 +226,10 @@ case class CompletionProposal(
         && (!overwrite || !paramsProbablyExists)
         && explicitParamNames.flatten.nonEmpty)
 
+      val charactersAdded = completionFullString.length-(offset-startPos)
+      ScalaPlugin().statistics.incUses(CharactersSaved, numToInc = charactersAdded)
+      importStmt foreach (_ ⇒ ScalaPlugin().statistics.incUses(ImportMissingMember))
```

Comment:
I didn't get this line at first glimpse, just after a while I realized that `incUses` increments some counter